pi5/config.go

```diff
@@ -11,22 +9,10 @@ import (
 
 // A Config describes the configuration of a board and all of its connected parts.
 type Config struct {
-	AnalogReaders     []mcp3008helper.MCP3008AnalogConfig `json:"analogs,omitempty"`
-	DigitalInterrupts []board.DigitalInterruptConfig      `json:"digital_interrupts,omitempty"`
 }
```

Comment:
This empty struct is useless. Any time it would have been used, you could instead use `resource.TrivialConfig`. Can we delete it outright (and its `Validate` function, below)?

Reply:
I have never heard of `resource.TrivialConfig` and I don't see it at all in RDK. are you referring to `resource.TriviallyValidateConfig`? I can use that to get rid of the validate function, but I think all drivers need the config struct

Reply:
Sorry, I meant `resource.NoNativeConfig`, defined in the same file as `TriviallyValidateConfig`. I don't feel strongly; feel free to keep this here if you want. :shrug:

Reply:
I will keep it here since we will likely add a configuration for pull up/pull down in the future